app_publisher/README.md

```diff
@@ -14,11 +14,13 @@ This plugin publishes ROS topic at the beginning or end of the app.
   - `name`: name of the topic
   - `pkg`: package name of the message
   - `type`: type of the message
+  - `field`: content of the message field
 - `stop_topics`: topic which is published at the end of app
   - `name`: name of the topic
   - `pkg`: package name of the message
   - `type`: type of the message
   - `field`: content of the message field
+  - `cond`: Decide to publish a topic depending on the app exit condition. `succeed`, `fail` or `stop` can be set.
```

Comment:
I think `success`, `failure`, `stop` (noun) is better.

Reply:
Thank you very much for your review.
I updated.